Wrappers/Python/cil/optimisation/functions/SubsetSumFunction.py

```diff
@@ -182,48 +181,43 @@ class SAGAFunction(SubsetSumFunction):
         serves as diagonal preconditioner
         default None
     - (optional)
-        gradient_initialization_point: point to initialize the gradient of each subset
+        gradient_initialisation_point: point to initialize the gradient of each subset
         and the full gradient
         default None
            
     '''
 
-    def __init__(self, functions, precond=None, gradient_initialization_point=None, **kwargs):
+    def __init__(self, functions, precond=None, gradient_initialisation_point=None, **kwargs):
         
         super(SAGAFunction, self).__init__(functions)
 
-        if gradient_initialization_point is None:
```

Comment:
I moved this to the memory_init method. One could also do the memory check in the __init__?